ImageItem/ImageItem.module.less

```diff
@@ -0,0 +1,26 @@
+// ImageItem.module.less
+//
+
+@import '../styles/skin.less';
+@import '../styles/mixins.less';
+
+.imageItem {
+  	.applySkins({
+		.caption {
+	  		height: @agate-imageitem-caption-height;
+	  		line-height: @agate-imageitem-caption-height;
+	  		text-align: center;
+		}
+
+		&.horizontal {
+	  		.image {
+				margin: auto 0; // leaving this to align image with caption when orientation="horizontal"
+	  		}
+		}
+
+		.focus({
+	  		border: @agate-imageitem-border-width solid @agate-imageitem-border-color;
+	  		background-color: @agate-imageitem-focus-bg-color;
+		})
+  	});
```

Comment:
Some spaces and tabs are mixed as indentation. 2 spaces and 1 tabs. Please check all indentation in this file.

```
  	});
```

Reply:
I fixed it.